website/docs/cli-ref.md

```diff
@@ -233,6 +233,8 @@ Usage:
                                 Enable experimental LSP feature: Signature Help
       --enable-experimental-lsp-extract-to-variable
                                 Enable experimental LSP feature: Extract To Variable
+      --enable-experimental-lsp-multiple-dir
+                                Enable experimental LSP feature: Multiple --dir options
```

Comment:
Please make it clear that virtually no work has been done to fix the known problems preventing Sorbet from robustly supporting multiple input directories in LSP mode, and that using this flag does nothing more than silence the error which would otherwise make multiple directories completely unsupported.

Unlikely other experimental LSP features, you can mostly use the feature as is an expect things to work but that is not the case for this feature.

I would honestly go so far as to name this flag something other than `--enable-experimental-lsp-*`, something like "force allow multiple directories" or "forcibly silence multiple directories error" or something like that. If we were eventually to implement robust support for multiple directories, it's unlikely we would guard it behind an LSP experimental flag for a slow rollout—we would just ship it—so adhering to the same name for the flag doesn't make sense.

Also, this would more accurately reflect that the `enable-all-experimental-lsp-features` does not apply to this one (intentionally).

Reply:
Okay, I'll change this flag to `--forcibly-silence-multiple-directories-error`.